### ISSUE TITLE
[Chips] ChipsTypicalUseController chips are missing traits

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#3189]
### Overview
The ChipsTypicalUseViewController allow selection of chips. However, none of the chips have any accessibilityTraits or hints assigned.

### Reproduction steps
1. Open the catalog &rarr; Chips &rarr; Demo
1. Navigate the Chips in the view with VoiceOver

**Expected Result**
1. Each Chip has .button accessibilityTrait
1. Selected Chip has .selected accessibilityTrait
1. Each Chip has a hint with something like, "Selects this chip"
1. Delete button has accessibility label, "Delete"
1. Delete button has accessibility hint, "Deletes this chip", 

**Actual Result**
1. Each chip is has no traits (it is a label)
1. Selected chip is missing any traits for selection
1. Chips have no hints
1. Delete buttons have no label
1. Delete buttons have no hint

### Version number
50.0.0

---
#### Internal data
- Associated internal bug: [b/117179087](http://b/117179087)

**Comment by jverkoey** [comment]
Related to https://github.com/material-components/material-components-ios/issues/4314

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
